govuk_chat_evaluation/output_guardrails/evaluate.py

```diff
@@ -30,55 +32,113 @@ def classification(self) -> str:
                 return "false_negative"
 
     def for_csv(self) -> dict[str, Any]:
-        return {**self.model_dump(), "classification": self.classification}
+        return {**self.model_dump(), "classification": self.classification_triggered}
 
 
 class AggregateResults:
     def __init__(self, evaluation_results: List[EvaluationResult]):
         self.evaluation_results = evaluation_results
         counter = Counter(
-            evaluation_result.classification for evaluation_result in evaluation_results
+            result.classification_triggered for result in evaluation_results
         )
         self.true_positive = counter.get("true_positive", 0)
         self.true_negative = counter.get("true_negative", 0)
         self.false_positive = counter.get("false_positive", 0)
         self.false_negative = counter.get("false_negative", 0)
 
+        guardrail_set = {
+            name
+            for result in evaluation_results
+            for name in list(result.expected_guardrails.keys())
+            + list(result.actual_guardrails.keys())
+        }
+        self.guardrail_names: List[str] = sorted(guardrail_set)
+
     @cached_property
     def _expected_actual_lists(self) -> tuple[List[int], List[int]]:
-        pairs_list = [
-            (
-                int(evaluation_result.expected_outcome),
-                int(evaluation_result.actual_outcome),
+        expected, actual = zip(
+            *(
+                (
+                    int(evaluation_result.expected_triggered),
+                    int(evaluation_result.actual_triggered),
+                )
+                for evaluation_result in self.evaluation_results
             )
-            for evaluation_result in self.evaluation_results
-        ]
-        expected, actual = zip(*pairs_list)
+        )
```

Comment:
I'm not too sure why we changed this? it's probably a little easier to understand in it's previous form.